src/SquareRoots.java

```diff
@@ -0,0 +1,21 @@
+import java.util.Scanner;
```

Comment:
Please move this file to a different PR, and only keep to the exercism task 

One PR should include files for one topic. It's a bad habbit to include many different files in one PR.